local-dev/stack/config.yaml

```diff
@@ -72,3 +72,4 @@ FEATURE_QUOTA_MANAGEMENT: True
 BROWSER_API_CALLS_XHR_ONLY: False
 CORS_ORIGIN: "http://localhost:9000"
 FEATURE_UI_V2: True
+V2_API_PATH: '/v2_alpha'
```

Comment:
nit: can this be `/v2-alpha` instead? I think most conventions favor a hyphen in a URL